examples/nvcontrol.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python
+#
+# examples/nvcontrol.py -- demonstrate the NV-CONTROL extension
+#
+#    Copyright (C) 2019 Roberto Leinardi <roberto@leinardi.com>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public License
+# as published by the Free Software Foundation; either version 2.1
+# of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the
+#    Free Software Foundation, Inc.,
+#    59 Temple Place,
+#    Suite 330,
+#    Boston, MA 02111-1307 USA
+
+
+# Python 2/3 compatibility.
+from __future__ import print_function
+
+import sys
+import os
+
+# Change path so we find Xlib
+from pprint import pprint
+
+from Xlib.display import Display
+from Xlib.ext.nvcontrol import Gpu, Cooler
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+
+if __name__ == '__main__':
+    display = Display()
+    # Check for extension
+    if not display.has_extension('NV-CONTROL'):
+        sys.stderr.write('%s: server does not have the NV-CONTROL extension\n'
+                         % sys.argv[0])
```

Comment:
String interpolation with `%` syntax should be replaced by `.format()` usage as it's recommended way.

Reply:
Same as the previous comment (code took from `xinerama.py`)